### PR TITLE
rename(cli): --inspect-brk to --inspect-break

### PR DESCRIPTION
**Pull request by timreichen**
ref: https://github.com/denoland/deno/issues/9344

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -1022,7 +1022,7 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .takes_value(true)
         .requires("unstable")
         .conflicts_with("inspect")
-        .conflicts_with("inspect-brk")
+        .conflicts_with("inspect-break")
```
Comment (caspervonb) [inline cli/flags.rs:1025]: Not aliased? so breaking change.

**Review by caspervonb — CHANGES_REQUESTED** [review]
Am indifferent as to if we should land this or not, I'll punt that to the others but it has to be a non-breaking change to land for 1.x.

File: cli/flags.rs (hunk 1 above)
Comment (timreichen) [reply cli/flags.rs:1025]: I am not too familiar with rust and would need help on how exactly to implement aliases. Else this could also wait till 2.x

Comment (nayeemrmn) [reply cli/flags.rs:1025]: See https://github.com/denoland/deno/pull/7032/files#diff-abdd2d887e97d0eff6d352a1efd4dd953797699228a18a3a237d9818469da3cfR1326 for an example.

**Comment by kitsonk** [comment]
Yes, this needs to be a non-breaking change to be considered, even then `--inspect-brk` while poorly named, reflects the same flag across all the Chrome based inspectors, so it matches a wider set of expectations.

File: cli/flags.rs (hunk 1 above)
Comment (timreichen) [reply cli/flags.rs:1025]: added alias

**Comment by bartlomieju** [comment]
Discussed with the rest of the team, and I whole-heartedly agree with @kitsonk's [comment](https://github.com/denoland/deno/issues/9344#issuecomment-826394772). This flag name was chosen before 1.0 specifically to match flag name in Node and Chrome. 

So thank you @timreichen for the PR, but I'm going to close it without a merge.

**Pull request closed without merging**